var/spack/repos/builtin/packages/timemory/package.py

```diff
@@ -8,30 +8,56 @@
 from spack import *
 
 
-class Timemory(CMakePackage):
-    """Timing + Memory + Hardware Counter Utilities for C/C++/CUDA/Python"""
+class Timemory(CMakePackage, PythonPackage):
```

Comment:
WRT my concern about inheriting multiple build systems, I don't think that's an issue: apparently if two inherited classes define the same method, the first is used.